test_run.lua

```diff
@@ -40,8 +40,12 @@ local function get_param(self, node, param)
 end
 
 local function get_lsn(self, node, sid)
-    local nodes = self:get_param(node, 'vclock')
-    return tonumber(nodes[1][tonumber(sid)])
+    local node = self:get_param(node, 'vclock')[1]
```

Comment:
The terminology was broken before your patch, but, please, fix it. Otherwise the code will be confusing as for you as well as for others.

It is not a 'node' (a vclock component). It is the whole vclock. `test_run:cmd([[eval foo "return 42"]])` gives a table as result (`{42}`), because in the general case evaluation of a Lua statement may give several values as result (so called multireturn).

Reply:
Ok, I see, rolled-back.